code/game/machinery/computer/crew.dm

```diff
@@ -33,9 +33,17 @@
 		src.updateDialog()
 		return
 
+var/datum/singleton/crewmonitor/crewmonitor = new

```

Comment:
I added the word "singleton" to the type path, not the variable declaration. The idea is that it's immediately obvious that it's only used once, without even being aware there's a global variable with the same name.

I've changed the variable definition, but I'd rather not get rid of the word "singleton" in the type path. Is this ok?